apiclient/response_handlers.py

```diff
@@ -23,6 +27,34 @@ class RequestsResponseHandler(BaseResponseHandler):
     def get_request_data(response: Response) -> Response:
         return response
 
+    @classmethod
+    def get_exception_class(cls, response: Response) -> Optional[Type[Exception]]:
+        return None
+
+
+class RequestsExceptionByKeyMixin:
```

Comment:
This should be necessary if we remove everything except the exception handler on the main client.

Reply:
I didn't get it. Remove it?

Reply:
👌